pkgs/development/python-modules/aesara/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, scipy
+, filelock
+, etuples
+, logical-unification
+, minikanren
+, cons
+, numba
+, numba-scipy
+, libgpuarray
+, sympy
+, cython
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "aesara";
+  version = "2.4.0";
+
+  src = fetchFromGitHub {
+    owner = "aesara-devs";
+    repo = "aesara";
+    rev = "38d7a813646c1e350170c46bafade0e7d0e2427c";
+    sha256 = "sha256-933bM15BZi4sTjnIOGAg5dc5tXVWQ9lFzktOtzj5DNQ=";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    scipy
+    filelock
+    etuples
+    logical-unification
+    minikanren
+    cons
+    numba
+    numba-scipy
+    libgpuarray
+    sympy
+    cython
+  ];
+
+  # Test failing with: `collect2: error: ld returned 1 exit status`
```

Comment:
Yes, I included the full log as a comment in the PR

Reply:
I think the problem is related to this line: ```/homeless-shelter/.aesara/compiledir_Linux-5.4--generic-x86_64-with-glibc2.33--3.9.10-64/tmpl3kgilx3/mfd7e6d3547b91eb16dd42369ba7533d8a8769a2ee66a413b9ed97a60305d8231.so' used in link```

That file shouldn't exist because ``/homeless-shelter`` does not exist. I think you want to set ``$HOME`` to a temporary directory before the build step.

```nix
preBuild = ''
  export HOME=$(mktemp -d)
'';
```

Reply:
Works great, thanks